addons/utm/models/utm_stage.py

```diff
@@ -0,0 +1,16 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+
+from odoo import fields, models
+
+
+class UtmStage(models.Model):
+    """Stage for utm campaigns."""
+
+    _name = 'utm.stage'
+    _description = 'Campaign Stage'
+    _order = 'sequence'
+
+    name = fields.Char(required=True, translate=True)
+    sequence = fields.Integer()
```

Comment:
Shouldn't we add a default on sequence of stages, while we are at it ? Better have duplicated sequences than a bunch of False.